### DISCUSSION TITLE
Are supported Tizen 2.3 and 2.4 Smart TV platforms ?

### DISCUSSION THREAD

**Discussion by jeremies** [necolas/react-native-web#2480] [Q&A]
Are supported Tizen 2.3 and 2.4 Smart TV platforms ? Does anyone has tried to build an app with react-native-web with these platforms?

**Comment by Dovakeidy** [comment]
The issue does not seem to be related to react-native-web. This library is meant to bring react-native.. to the web, not to the Smart TV. I think your question should be asked on this repository: https://github.com/react-native-tvos/react-native-tvos .

**Reply by jeremies** [reply]
Sorry, this issue is related to react-native-web, since many Smart TVs can show web apps that run in a browser including Tizen. Tizen 2.3 and Tizen 2.4 run in a type of webkit web engine: https://developer.samsung.com/smarttv/develop/specifications/web-engine-specifications.html

**Reply by jeremies** [reply]
Does anybody knows if react-native-web runs OK in old chromium or webkit browsers? Which could be the stoppers?
